### PR TITLE
Header

### PR DESCRIPTION
**Pull request by NataliyaKoval**

### CODE REVIEW

File: src/app/modules/core/components/burger-menu/burger-menu.component.html [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+  </symbol>
+
+  <symbol fill="none" id="arrow-down" viewBox="0 0 24 24">
+    <path clip-rule="evenodd" d="M6 9.293l6 6 6-6 .707.707L12 16.707 5.293 10 6 9.293z" fill="#0E0C08"
+          fill-rule="evenodd"/>
+  </symbol>
+</svg>
+
+<div (click)="closeBurgerMenu()" class="page-container">
+  <div (click)="$event.stopPropagation()" class="burger-wrapper">
```
Comment (yana-tsylinska) [inline src/app/modules/core/components/burger-menu/burger-menu.component.html:36]: replace with window listener - https://stackoverflow.com/questions/51150422/how-to-detect-click-outside-of-an-element-in-angular

File: src/app/modules/core/components/burger-menu/burger-menu.component.html [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+          <li><a class="sub-link" routerLink="/persons/administration">
+            Administration
+          </a>
+          </li>
+        </ul>
+      </li>
+
+      <li (click)="showSubMenuAbout = !showSubMenuAbout" class="menu-item">
+        <div class="li-container">About
+          <svg [class.opened]="showSubMenuAbout" height="24" width="24">
```
Comment (yana-tsylinska) [inline src/app/modules/core/components/burger-menu/burger-menu.component.html:80]: add link here and show submenu when link is active

**Review by yana-tsylinska — CHANGES_REQUESTED** [review]

**Review by spolischook — APPROVED** [review]
Hide menu when user click on menu item

File: src/app/modules/core/components/header/header.component.html [hunk 3]
```diff
@@ -1,44 +1,136 @@
...
+         class="language ukr">Укр&nbsp;</a> <span class="separation">|</span>
+      <a [href]="langRedirectUrl" class="language eng">&nbsp;Eng</a>
+
+      <a class="header-button btn tickets-button"
+         href="https://widget.mticket.com.ua/uk/widget332site11795/widget/index"
+         i18n="@@buyTicket" target="_blank">Buy ticket</a>
+    </div>
+
+    <div *ngIf="!wideScreen" class="icons-group">
+      <svg (click)="this.donateService.activeDonateMenu()" class="icon" height="36" width="36">
```
[the remark this answers is not in the document]
Comment (mAndrushchenko) [reply src/app/modules/core/components/header/header.component.html]: Fixed

File: src/app/modules/core/components/header/header.component.html [hunk 4]
```diff
@@ -1,44 +1,136 @@
...
+        <use href="#wallet"></use>
+      </svg>
+
+      <svg (click)="trigger.next(true)" class="icon" height="36" width="36">
+        <use href="#eye"></use>
+      </svg>
+
+      <svg (click)="burgerMenuIsOpened = !burgerMenuIsOpened" class="icon" height="36" width="36">
```
[the remark this answers is not in the document]
Comment (mAndrushchenko) [reply src/app/modules/core/components/header/header.component.html]: Fixed

File: src/app/modules/core/components/header/header.component.html [hunk 5]
```diff
@@ -1,44 +1,136 @@
...
+        <use href="#burger"></use>
+      </svg>
```
[the remark this answers is not in the document]
Comment (mAndrushchenko) [reply src/app/modules/core/components/header/header.component.html]: Fixed

File: src/app/modules/core/components/header/header.component.ts [hunk 6]
```diff
@@ -1,33 +1,56 @@
...
+  @HostListener('window:resize', ['$event'])
+  onResize() {
+    this.getWindowSize();
+  }
+
+  getWindowSize() {
+    const screenWidth = window.innerWidth;
+    const wideScreen = 1240;
+
+    this.wideScreen = screenWidth > wideScreen;
```
[the remark this answers is not in the document]
Comment (mAndrushchenko) [reply src/app/modules/core/components/header/header.component.ts:52]: Fixed

File: src/app/modules/core/components/burger-menu/burger-menu.component.html (hunk 1 above)
Comment (mAndrushchenko) [reply src/app/modules/core/components/burger-menu/burger-menu.component.html]: Fixed

File: src/app/modules/core/components/burger-menu/burger-menu.component.html (hunk 2 above)
Comment (mAndrushchenko) [reply src/app/modules/core/components/burger-menu/burger-menu.component.html]: Fixed

**Pull request merged**